FinSearchUnified/Tests/Bundle/SearchBundleFindologic/ConditionHandler/ProductAttributeConditionHandlerTest.php

```diff
@@ -70,10 +71,10 @@ public function attributesDataProvider()
                 ],
                 ['color' => ['' => ['blue', 'red']]]
             ],
-            'Vendor is "Brand+Name" and color is "red"' => [
+            'Vendor is "Brand Name" and color is "red"' => [
                 Operator::EQ,
                 [
-                    ['field' => 'vendor', 'value' => 'Brand+Name'],
```

Comment:
Was the this case testing the wrong thing? When encoded twice this would result in `Brand%2BName`.

Reply:
It is more like this:
1. User submits filter. URL contains `...filterName=Brand%2BName`
2. When Shopware builds their `Criteria` event, their `Request` class already contains all query related information URL decoded, therefore this value then is `Brand+Name`. This is already done at a PHP level, where PHP populates the superglobal `$_GET` with query information which is **already url decoded**.
3. In our old logic we then took `Brand+Name` and url decoded it again, so it will be `Brand Name`, which is a completely different filter.